components/http/okHttp/gradle/dependencies.gradle

```diff
@@ -12,6 +12,7 @@ dependencies {
     implementation 'io.opentelemetry:opentelemetry-api:1.41.0'
     implementation 'io.opentelemetry:opentelemetry-context:1.41.0'
     implementation 'jakarta.annotation:jakarta.annotation-api:2.1.1'
+    implementation 'com.google.guava:guava:33.3.0-jre'
```

Comment:
some context you might not have: we explicitly removed guava from the dependencies with v6 to avoid multiple issues it was causing (https://github.com/microsoftgraph/msgraph-sdk-java/issues/1860 and others)
We should probably look at a solution that does not involve bringing back guava